Lib/test/test_asyncio/test_events.py

```diff
@@ -1145,6 +1151,7 @@ def test_legacy_create_unix_server_ssl_verify_failed(self):
             self.test_create_unix_server_ssl_verify_failed()
 
     @unittest.skipIf(ssl is None, 'No ssl module')
+    @unittest.skipIf(IS_OPENSSL_1_1, "bpo-26470: fail on OpenSSL 1.1")
```

Comment:
Is the version number 1.1 and not 1.1.0 here by accident or purpose?

```suggestion
    @unittest.skipIf(IS_OPENSSL_1_1, "bpo-26470: fail on OpenSSL 1.1.0")
```

That would make it more consistent.

Also, have we checked that it's actually 1.1.0 and not 1.1.1? Should we say 1.1 everywhere instead?

Reply:
> Also, have we checked that it's actually 1.1.0 and not 1.1.1?

No, and I'm not interested to test if it's exactly 1.1.0 or 1.1.1. I'm tired of the OpenSSL 1.1.1 mess, I consider that I already spent enough time on this topic :-)

> Should we say 1.1 everywhere instead?

I fixed the test_asyncio comment to write OpenSSL 1.1.0, as I did in test_ssl.

Reply:
ok

Reply:
For the record, the tests pass with OpenSSL 1.1.0i.